src/audio.c

```diff
@@ -117,61 +117,29 @@ struct autx {
 };
 
 
-/**
- * Audio receive/decoder
- *
- \verbatim
-
- Processing decoder pipeline:
-
-       .--------.   .-------.   .--------.   .--------.
- |\    |        |   |       |   |        |   |        |
- | |<--| auplay |<--| aubuf |<--| aufilt |<--| decode |<--- RTP
- |/    |        |   |       |   |        |   |        |
-       '--------'   '-------'   '--------'   '--------'
-
- \endverbatim
- */
 struct aurx {
```

Comment:
is it possible to refactor the whole struct ?

Reply:
The fields that not moved to `struct audio_recv` have no r/w with the RX thread.

Reply:
I try to move them also to `struct audio_recv` in aureceiver.c